src/main/public/stylesheets/components/_web-chat.scss

```diff
@@ -3,3 +3,16 @@
     margin-bottom: 0;
   }
 }
+
+.heading-medium,
```

Comment:
this will change the heading-medium across all of the site. I don't think that is what you're trying to do?